documentation/final-deliverables/pdf-report.md

```diff
@@ -105,16 +105,55 @@ FR23. To "unmatch" Mentees and Mentors, the Mentee can communicate with the Ment
 
 ## Technical Specifications
 
- (needs to be added)
+**Overview:**
+
+Although the decision for our chosen tech stack was ultimately up to us, our client has technical experience, and so she offered recommendations for front-end and back-end libraries back in September. As such, we aimed to align with the client’s technical suggestions, as well as to pick an MVC model which was well documented. Ultimately, this resulted in React being used for our front-end, Node for our back-end, and Firebase for our database.
+
+**Technical Specifications on the Front-end:**
+
+- React is a library that utilizes JavaScript to create user interfaces.
+- Code pertaining to the front-end can be accessed by navigating to the ‘src’ folder, then by clicking on the ‘front-end’ folder.
+- ‘App.js’ is the main page that launches upon the application being run.
+  - Every other UI page is split into components that are rendered dependent on the program’s state from the user’s actions. Given this, all of our UI components exist in their own folder named according to what it displays within the central folder titled ‘components’.
+- ‘helper-methods.js’ contains various functions that are called throughout the application. The majority of these functions play a crucial role for interacting with the Firebase database (for example, to create users after form submission, to send verification and password reset emails for authenticated users, etc.).
+- Credentials for interacting with the database are within firebase-init.js.
+- ‘index.js’ allows the Mentee and Mentor forms to be rendered independently from the central website.
+- ‘timeZones.json’ is utilized to populate the ‘time zone’ dropdown on the Mentee and Mentor forms.
+
+A major benefit to using React is the ability to use UI Libraries (we opted for Ant Design), as well as to integrate with our back-end Node server. The main Ant Design components utilized throughout the app are Notifications, Cards, Forms, Icons, Tabs, and Modals.
+
+**Technical Specifications on the Back-end:**
+
+- Node is a back-end environment which uses JavaScript.
+- Our back-end is accessed from the ‘back-end’ folder within ‘src’.
+- ‘App.js’ contains functions that are triggered from events on a specific port. For example, functions exist for generating matches between Mentees and Mentors (by finding the least difference in time zones, number of commons skills, and same organization string) as well as route calls for inviting new mentors and accepting mentees.
+- ‘Email-helpers.js’ houses functions that send custom emails to users via nodemailer.
+- The back-end is configured to use the Firebase Admin SDK as a way of securing user information.
```

Comment:
```suggestion
- The back-end is configured to use the Firebase Admin SDK. This is needed to give our backend full access to our firebase database, which is, otherwise, secured by rules. 
```